### PR TITLE
Make github wiki links relative for website use

### PR DESCRIPTION
**Pull request by ChrisMaddock**
I use the website for the docs rather than the github wiki. Occasionally, I click on a link and it takes me to a wiki page, rather than a website page. I presume this is accidental and all these links should be relative, so did a quick find and replace while I had the repo open - feel free to close this one if I've missed the point!

### CODE REVIEW

File: CommunityPackagesDisclaimer.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ Support is not guaranteed by package maintainers, but should you encounter issue
...
+For folks in the community that are not using Chocolatey for production purposes, with an understanding of the disclaimers set forth here, it is fine to use the community repository. For organizations that have a low tolerance for breakages and require a higher level of reliability, security, trust, and control, a self-hosted Chocolatey server is the recommended option. This guarantees that your installs, upgrades, and uninstalls will always work every time. This gives you complete control over what software gets installed. Also because you are vetting newer versions, you control when those newer versions are available for upgrade. Please see [Host Your Own Server](How-To-Host-Feed) for more information.
```
Comment (ferventcoder) [inline CommunityPackagesDisclaimer.md]: Shouldn't this be [[Host Your Own Server|How-To-Host-Feed]] ? I'm sure that likely works as is though.

Comment (ChrisMaddock) [inline CommunityPackagesDisclaimer.md]: Sorry - I'd done them how I was used to, missed the convention! Happy to update them if you'd prefer?

Comment (ChrisMaddock) [inline CommunityPackagesDisclaimer.md]: Is the `[[ | ]]` formatting necessary for the website generation?

Comment (ferventcoder) [inline CommunityPackagesDisclaimer.md]: Not sure, I'm going to verify when I pull in the next time to see for sure.

Comment (ferventcoder) [inline CommunityPackagesDisclaimer.md]: It worked fine!

Comment (ferventcoder) [inline CommunityPackagesDisclaimer.md]: Thanks for the help!

Comment (ChrisMaddock) [inline CommunityPackagesDisclaimer.md]: Great - thanks! :-)

**Pull request merged**